doc/sphinx/practical-tools/coqide.rst

```diff
@@ -330,3 +330,182 @@ If you choose something else than UTF-8, then missing characters will
 be written encoded by `\x{....}` or `\x{........}` where each dot is
 an hexadecimal digit: the number between braces is the hexadecimal
 Unicode index for the missing character.
+
+.. _coqide-debugger:
+
+Debugger
+--------
+
+Version 8.15 introduces a visual debugger for |Ltac| tactics within
+CoqIDE.  It supports setting breakpoints visually and automatically
+displaying the stopping point in the source code with "continue",
+"step over" "step in" and "step out" operations.  The call stack and variable
+values for each stack frame are shown in a new panel.
+
+The debugger is based on the non-visual |Ltac| :ref:`debugger <interactive-debugger>`.
+We'd like to eventually support other scripting facilities such as Ltac2.
+
+Since the visual debugger is new, you may encounter bugs or usability issues.
+The behavior and user interface will evolve as the debugger is refined.
+There are notes on bugs and potential enhancements at the end of
+`this page <https://github.com/coq/coq/wiki/Ltac-Debugger-Preview>`.
+Feel free to suggest changes and improvements by opening an issue on
+`GitHub <https://github.com/coq/coq/issues/new>`_, or contact `@jfehrle`
+directly through email, Zulip or Discourse.
+
+Breakpoints
+~~~~~~~~~~~
+
+This screenshot shows the debugger stopped at a breakpoint in the |Ltac| tactic
+`my_tac`.  Breakpoints are shown with a red background and the stopping point is
+shown with a dark blue background.  `Set Ltac Debug.` enables stopping in the
+debugger.
+
+  .. image:: ../_static/debugger.png
+     :alt: CoqIDE Debugger
+
+You can control the debugger with function and control keys.  Some
+messages are shown in the Messages panel.  You can type
+:ref:`debugger commands <interactive-debugger>`
+in that panel when it shows the debug prompt.
+
+The script is not editable while Coq is processing tactics or stopped
+in the debugger.  When Coq is stopped in the debugger (e.g., at a breakpoint),
+the blue segment in the "in progress" slider at the bottom edge of the window
+will be stopped at the left hand edge of its range.
+
+The function keys are listed, for the moment, with one exception, in the `Debug` menu:
```

Comment:
I'll update the screen shot.  The challenge with screen shots is that they can quickly become a bit out of date.  Noticing that and updating the images is a bit tedious: you have to recreate the .v file used to create it and often it takes a couple tries to get all the right steps (e.g. set two breakpoints and expand "con" in the variables).  I don't think we should try to update them for every minor GUI tweak.

Also, BTW, the appearance is a bit platform-specific.  For example, the icons are different in the Windows native version (this is 8.13 below).  It would be good if we provided some of our own icons so it looks more consistent across platforms.  That would probably mean hiring someone to create them--not my skill set.

![image](https://user-images.githubusercontent.com/1253341/142055416-0bde9100-eb3e-4303-8650-546d7537e2c8.png)

Reply:
I agree that it is not important to keep screenshots up-to-date. But this is a case where it made sense to update it because you talk about a Debug menu, so we expect to see it in the screenshot just above.